Astro CLI v1.0.0 (All Changes) (#690)
astronomer/docs: software/integrate-iam.md

* All pods within your Airflow Deployment will assume the IAM role. There is currently no way to use more than 1 IAM role per deployment.
* If you’d like your IAM role to apply to more than 1 deployment, you must annotate each deployment.
* You must use the Astro CLI to pass IAM role annotations.
* Only Workspace Admins can pass IAM role annotations.

> * Once a Deployment is created or updated with an IAM role, there is no way to delete that annotation.

Reviewer: ```suggestion
* Once a Deployment is created or updated with an IAM role, the annotation can't be deleted.
```

## Prerequisites

* [The Astro CLI](cli-quickstart.md)
* Admin access on an Astronomer Workspace

> * Direct access to your Kubernetes Cluster (e.g. permission to run `$ kubectl describe po`)

Reviewer: ```suggestion
* Direct access to your Kubernetes Cluster (for example, permission to run `$ kubectl describe po`)
```
